exitdir_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2022 Chainguard, Inc.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package exitdir_test
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"chainguard.dev/exitdir"
+)
+
+func TestAwareExit_empty(t *testing.T) {
+	if err := os.Setenv("EXIT_DIR", t.TempDir()); err != nil {
+		t.Fatal(err)
```

Comment:
You can use the `Example_` style tests to turn some of these into examples in the resulting `godoc`, which would be awesomesauce.